src/Controllers/DatabaseController.js

```diff
@@ -360,6 +360,15 @@ const relationSchema = {
   fields: { relatedId: { type: 'String' }, owningId: { type: 'String' } },
 };
 
+const maybeTransformUsernameAndEmailToLowerCase = (object, className, options) => {
```

Comment:
- remove `maybe`
- use one term across all code, currently we have "force" and "transform", let's use "convert" for all

Reply:
we can remove `maybe` but i'll keep transform. It's the term used in this file


```js
              update = transformObjectACL(update);
              transformUsernameAndEmailToLowerCase(update, className, this.options);
              transformAuthData(className, update, schema);
```

Reply:
Which file? A "transformation" implies a more complex change of structure, this is simply a case conversion. Please use convert.